build.cake

```diff
@@ -36,6 +36,7 @@ var GUI_TESTS = BIN_DIR + "nunit-gui.tests.dll";
 // Package sources for nuget restore
 var PACKAGE_SOURCE = new string[]
 {
+    "https://www.nuget.org/api/v2",
```

Comment:
This comes up as different people (or the same person at different times) use different editors. We should fix when we are working in the file again but it's not a big priority.